CMakeLists.txt

```diff
@@ -90,6 +90,17 @@ check_function_exists(getspnam HAVE_GETSPNAM)
 
 # XAU
 pkg_check_modules(LIBXAU REQUIRED "xau")
+# If possible, convert xau to a full path; if not, keep the existing
+# setting (which is probably -lXau .. which fails on FreeBSD because
+# the **directory** it is in is not searched.
+find_library(_xau_lib
```

Comment:
According to [docs](https://cmake.org/cmake/help/latest/module/FindPkgConfig.html), `LIBXAU_LINK_LIBRARIES` should contain libraries with absolute paths. Doesn't that work?